docs/src/pages/styles/basics/basics.md

```diff
@@ -8,7 +8,7 @@ You can use it, but you don't have to, since Material-UI is also [interoperable
 
 ## Why use Material-UI's styling solution?
 
-In previous versions, Material-UI has used LESS, then a custom inline-style solution to write the component styles, but these approaches have proven to be limited.
+In previous versions, Material-UI has used LESS than a custom inline-style solution to write the component styles, but these approaches have proven to be limited.
```

Comment:
```suggestion
In previous versions, Material-UI has used [Less](http://lesscss.org/), and then a custom inline-style solution to write the component styles, but these approaches proved to be limited.
```

Reply:
LOL. Yep that makes a lot more sense. Thanks for clarifying that!